crates/egui_extras/src/table.rs

```diff
@@ -519,6 +525,18 @@ pub struct Table<'a> {
     cell_layout: egui::Layout,
 
     scroll_options: TableScrollOptions,
+    /// The `header_response` is passed down from [`TableBuilder::header()`] so that it can be returned to the user after the body is built.
```

Comment:
```suggestion

    /// The `header_response` is passed down from [`TableBuilder::header()`] so that it can be returned to the user after the body is built.
```